Dockerfile

```diff
@@ -20,9 +20,10 @@ COPY . .
 RUN mkdir -p audio transcripts
 
 # Set environment variables
+ENV APP_ENV=production
```

Comment:
This also configures Sinatra to `bind` to `0.0.0.0`, ie. allow access from any IP. For our use case (Sinatra as a web app), it makes more sense to have access control / interface binding done at the network configuration level, rather than application config.